legal-api/report-templates/template-parts/certificate-of-name-change/registrarSignature.html

```diff
@@ -0,0 +1,3 @@
+<div class="registrar-signature">
+  <img alt="signature" src={{registrarInfo.signature}}>
+</div>
```

Comment:
This looks like it would be a common file as well. Is it?

Reply:
Yes. Thanks.